portfolio/src/main/webapp/js/TestingReadMe.txt

```diff
@@ -0,0 +1,8 @@
+The tests in custom-script-spec.js require two dependencies I believe
+  The Jasmine framework for the testing.
+    npm install -g jasmine
+  The JSDom library required to utilisie document calls
+    npm install -g jsdom
+
+You need to remove the typewriterFeature() call from the global scope
+to run the tests.
```

Comment:
Add a TODO here to remove this when the typewriterFeature is removed from global scope.

Reply:
Sure Kenneth